src/interfaces/pyo3/py_cover_crypt.rs

```diff
@@ -256,7 +225,8 @@ impl CoverCrypt {
         public_key: &PublicKey,
         additional_data: Option<Vec<u8>>,
         authenticated_data: Option<Vec<u8>>,
-    ) -> PyResult<(SymmetricKey, Vec<u8>)> {
+        py: Python,
```

Comment:
argument description is missing

Reply:
PyO3 is automatically passing the `py` argument (https://pyo3.rs/main/doc/pyo3/marker/struct.python) so it's not an external input.

Reply:
Ok I did not know